lib/cisco_os_shim/core/version.rb

```diff
@@ -18,5 +18,5 @@ module Cisco
 
 # Add version number to Cisco::Shim namespace
 module Cisco::Shim
-  VERSION = '1.1.0'
+  VERSION = '1.0.0-dev'
```

Comment:
It's possible to rename a repo on GitHub, so yes I'd like to rename this repo from cisco-nxapi to cisco-os-shim (But note that requires approval by people outside our team).

This will be a new Gem release - we'll also need to push out an update to cisco_nxapi on rubygems.org to indicate that it's deprecated and replaced by cisco_os_shim-nxapi gem